package.xml

```diff
@@ -16,8 +16,11 @@
   <author>Felix Messmer</author>
   <author>Miguel Prada Sarasola</author>
   <author>Denis Stogl</author>
+  <author>Andy Zelenak</author>
+  <author>Marvin Grosse Besselmann</author>
 
-  <maintainer email="grosse@fzi.de">Marvin Große Besselmann</maintainer>
+  <maintainer email="exner@fzi.de">Felix Exner</maintainer>
+  <maintainer email="schnell@fzi.de">Tristan Schnell</maintainer>
   <maintainer email="lovro.ivanov@gmail.com">Lovro Ivanov</maintainer>
   <maintainer email="denis.stogl@picknik.ai">Denis Stogl</maintainer>
   <maintainer email="zelenak@picknik.ai">Andy Zelenak</maintainer>
```

Comment:
```suggestion
```

I don't think I need to be here anymore. Thanks for asking, though.

Reply:
Thanks for the quick answer!